File: 03-aritmetica/aritmetica.pas
Program Aritmetica;
Var
    a, b                      :  Real;
    suma, producto:           :  Real;
    cocienteAB, cocienteBA    :  Real;
Begin
    Write('Introduce un valor para a: ');
    Read(a);

    Write('Introduce un valor para b: ');
    Read(b);

    suma       := a + b;
    producto   := a * b;
    cocienteAB := a / b;
    cocienteBA := b / a;
    
    WriteLn('a + b = ' + suma);
    WriteLn('a * b = ' + producto);
    WriteLn('a / b = ' + cocienteAB);
    WriteLn('b / a = ' + cocienteBA);
End.

File: 03-aritmetica/aritmetica.hs
main :: IO ()
main = do
    putStrLn "Introduzca un valor para a: "
    aCadena <- getLine
    
    putStrLn "Introduzca un valor para b: "
    bCadena <- getLine

    let a = read aCadena :: Float
        b = read bCadena :: Float
        suma = (+) a b
        producto = a * b
        cocienteAB = (/) a b
        cocienteBA = b / a

    putStrLn $ "a + b = " ++ show suma
    putStrLn $ "a * b = " ++ show producto
    putStrLn $ "a / b = " ++ show cocienteAB
    putStrLn $ "b / a = " ++ show cocienteBA
